lib/ansible/modules/network/nxos/nxos_file_copy.py

```diff
@@ -100,7 +98,7 @@
     HAS_SCP = False
 
 
-def execute_show_command(command, module, command_type='cli_show'):
+def execute_show_command(command, module):
```

Comment:
The method has command type `cli_show_ascii` all the time it was called https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/nxos/nxos_file_copy.py#L114-L141 which means `text` as output.
I think it the output is `text` anyway by default. Correct me if I am wrong.
That's the reason to drop the parameter.